### ISSUE TITLE
NRE in CreateToolWindow

### ISSUE THREAD

**Issue by karthiknadig** [Microsoft/RTVS#3898]
For toolWindowGuid = {99d2ea62-72f2-33be-afc8-b8ce6e43b5d0}
```
System.NullReferenceException occurred
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=Microsoft.VisualStudio.R.Package
  StackTrace:
   at Microsoft.VisualStudio.R.Packages.R.RPackageToolWindowProvider.CreateToolWindow(Guid toolWindowGuid, Int32 id) in G:\GIT\RTVS\src\Package\Impl\Packages\R\RPackageToolWindowProvider.cs:line 29
   at Microsoft.VisualStudio.R.Packages.R.RPackage.CreateToolWindow(Guid& toolWindowType, Int32 id) in G:\GIT\RTVS\src\Package\Impl\Packages\R\RPackage.cs:line 186
   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.Shell.Interop.IVsToolWindowFactory.CreateToolWindow(Guid& toolWindowType, UInt32 id)
   at Microsoft.VisualStudio.Platform.WindowManagement.WindowFrame.ConstructContent()
```

**Comment by MikhailArkhipov** [comment]
Yeah, I fixed it already as part of #3771

**Comment by MikhailArkhipov** [comment]
https://github.com/Microsoft/RTVS/commit/235d2372fc802648005bd71d2291feaf7c2f84f4
